java/sbt/dse/src/main/java/com/datastax/spark/example/WriteRead.java

```diff
@@ -58,15 +56,19 @@ public Object apply(Session session)
       .cassandraTable("ks", "kv", mapRowToTuple(Integer.class, Integer.class))
       .collect();
 
-    // Read Data Using the Sql Context
-    sqlContext.setKeyspace("ks");
-    List<Tuple2<Integer, Integer>> sqlReadData = sqlContext.sql("SELECT * FROM kv").javaRDD()
+    // Read Data Using the Hive Context
+    List<Tuple2<Integer, Integer>> sqlReadData = hiveContext
```

Comment:
Without DSE the metastore won't exist, but you can make an entry by registering a DataFrame or by using the following syntax

https://github.com/datastax/spark-cassandra-connector/blob/master/doc/14_data_frames.md#creating-dataframes-using-spark-sql